### PR TITLE
changed field names

### PR DESCRIPTION
**Pull request by itay1313**
DX-32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ Jekyll uses a [front-matter](https://jekyllrb.com/docs/frontmatter/) block to ar
   - If the page does not have a `subfoldername` value: `documentname-pagename`. For example: `users-api-overview.html`.
 * `indicator`: This key contains a chat or messaging indicator (or both) for a document. It accepts `chat`, `messaging`, or `both` as its values.
 * `layout`: Sets the value to `hidden-layout`, if you want this file to be ignored in search engines.
+* `date_published`: This adds the date when it was published. `date_published: 2019/01/30` needs to be added manually
```
Comment (j9t) [inline README.md:147]: Suggestion:

```
`date_published`: This displays the date when the page was published (needs to be done manually). For example: `date_published: 2019/01/30`.
```

File: README.md [hunk 2]
```diff
@@ -144,6 +144,8 @@ Jekyll uses a [front-matter](https://jekyllrb.com/docs/frontmatter/) block to ar
...
+* `date_updated`: Last modified date added. Here's an example: `date_updated: 2022/01/30`
```
Comment (j9t) [inline README.md:148]: Suggestion:

```
`date_updated`: This defines and shows the date of the last substantial update. For example: `date_updated: 2022/01/30`.
```

**Review by j9t — CHANGES_REQUESTED** [review]
Great update! This looks really solid by now. Suggesting copy tweaks, make this more consistent with the other descriptions. (We can edit more later.)

File: README.md (hunk 2 above)
Comment (itay1313) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (j9t) [reply README.md:147]: I can still see the previous version—was the change submitted?

**Review by j9t — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (j9t) [reply README.md:147]: As mentioned, please make the information here consistent with the other lines, as with

```
`date_published`: This displays the date when the page was published (needs to be done manually). For example: `date_published: 2019/01/30`.
```

**Review by j9t — CHANGES_REQUESTED** [review]

File: _layouts/default.html [hunk 3]
```diff
@@ -2,66 +2,127 @@
 <html lang="en">
   <head>
     {% include head.html %}
-    <link rel="stylesheet" href="css/search.css">
-    <link rel="stylesheet" href="css/inner-page.css">
-    <meta name="description" content="LivePerson for developers: Build on the LivePerson Conversational Cloud with our powerful digital toolbox">
-    <meta name="keywords" content="APIs, SDKs, Mobile development, LiveEngage, Agent workspace, Tag, JSON examples, Click to message, Click to chat, Chat window, Skills, Agent Groups, iOS, Android, Mobile devices, Integrations, Live chat, Agent widget, Reporting, Analytics">
-    <meta name="author" content="LivePerson Inc.">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.css">
+    <link rel="stylesheet" href="css/search.css" />
```
Comment (j9t) [inline _layouts/default.html:5]: Sorry I missed this (was not expanded in the view):

Entire file: Check and correct formatting—for some reason, there are many inner-tag breaks.

(If this was intended and you believe we should change this everywhere, let me know—but it seems this was some tooling accident?)

**Review by j9t — CHANGES_REQUESTED** [review]

**Review by j9t — APPROVED** [review]

**Pull request merged**